Tests/js/views/services/plugins/assets/ez-locationsearchplugin-tests.js

```diff
@@ -253,9 +295,13 @@ YUI.add('ez-locationsearchplugin-tests', function (Y) {
             });
             Mock.expect(this.contentService, {
                 method: 'createView',
-                args: [this.query, Mock.Value.Function],
+                args: [Mock.Value.Object, Mock.Value.Function],
                 run: function (query, cb) {
-                    cb(false, that.response);
+                    if (query == that.locationQuery) {
```

Comment:
since you are comparing two objects, you should use `===`. Also because of that, none the condition should be evaluated to true so I'm a bit suspicious on that code and the test.